manifests/m/Microsoft/VisualStudio/2022/Community-Preview/17.0.31423.177/Microsoft.VisualStudio.2022.Community-Preview.installer.yaml

```diff
@@ -0,0 +1,53 @@
+﻿# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Microsoft.VisualStudio.2022.Community-Preview
+PackageVersion: 17.0.31423.177
+FileExtensions:
+  - asm
+  - asmx
+  - asp
+  - aspx
+  - c
+  - config
+  - cpp
+  - cppm
+  - cs
+  - csproj
+  - cxx
+  - h
+  - hpp
+  - hxx
+  - sln
+  - ts
+  - vcproj
+  - vcxproj
+  - xml
+Protocols:
+  - git-client
+  - vsls
+  - vssd
+  - vstfs
+  - vsweb
+Commands:
+  - devenv
+InstallerSuccessCodes:
+  - 1641
+  - 3010
+InstallModes:
+  - interactive
+  - silent
+  - silentWithProgress
+Installers:
+  - InstallerLocale: en-US
+    Architecture: x64
+    InstallerType: exe
+    Scope: machine
+    InstallerUrl: https://download.visualstudio.microsoft.com/download/pr/b6e51993-aee8-44a4-ab52-0fe6da4e98c6/7e61e637e06b5bc2d31174ba2a053e9d3b8cfd9a55669f1802905502086ca5b0/vs_Community.exe
+    InstallerSha256: 7E61E637E06B5BC2D31174BA2A053E9D3B8CFD9A55669F1802905502086CA5B0
+    InstallerSwitches:
+      Custom: --installPath <INSTALLPATH>
```

Comment:
Can you set the same Installer switches there were for VS 2019 here please (or the equivalent ones)? This doesn't install any workloads and will be pretty useless.

(I can track them down if needed but maybe you can get to it first?)

Reply:
Done! I've added the Microsoft.VisualStudio.Workload.Universal flag to match the other Visual Studio installers on all 3 PRs.